tests/konnector_structure.coffee

```diff
@@ -10,7 +10,9 @@ describe 'Check all konnectors', ->
     listOfKonnectors = fs.readdirSync konnectorsDirectory
     listOfLocales = fs.readdirSync localesDirectory
 
-    listOfKonnectors.forEach (filename) ->
+    # Remove hidden files (a.k.a. thoses starting with '.')
+    listOfKonnectors.filter (filename) -> filename.indexOf('.') != 0
```

Comment:
I considered that it would neither be a js/coffee file but this test could be more precise. `filename.charAt(0) != '.'` ?